libs/dataset-explorer/src/lib/TableView/TableViewDetailsHeader.tsx

```diff
@@ -10,22 +10,27 @@ import {
 } from "@fluentui/react";
 import React from "react";
 
-export const onRenderDetailsHeader: IRenderFunction<IDetailsHeaderProps> = (
-  props,
-  defaultRender
-) => {
-  if (!props) {
-    return <div />;
-  }
-  const onRenderColumnHeaderTooltip: IRenderFunction<IDetailsColumnRenderTooltipProps> =
-    (tooltipHostProps) => <TooltipHost {...tooltipHostProps} />;
-  return (
-    <div>
-      {defaultRender?.({
-        ...props,
-        onRenderColumnHeaderTooltip,
-        selectAllVisibility: SelectAllVisibility.hidden
-      })}
-    </div>
-  );
-};
+export function generateOnRenderDetailsHeader(
+  selectAllVisibility: SelectAllVisibility
+): IRenderFunction<IDetailsHeaderProps> {
+  const onRenderDetailsHeader: IRenderFunction<IDetailsHeaderProps> = (
+    props,
+    defaultRender
+  ) => {
+    if (!props) {
+      return <div />;
+    }
+    const onRenderColumnHeaderTooltip: IRenderFunction<IDetailsColumnRenderTooltipProps> =
+      (tooltipHostProps) => <TooltipHost {...tooltipHostProps} />;
+    return (
+      <div>
```

Comment:
nit: is this div needed?

Reply:
If I remove the div it seems to give the error "Parsing error: "," expected".
However, I changed the div to Stack and it was fine with it.